composer.json

```diff
@@ -4,8 +4,12 @@
   "type": "wordpress-plugin",
   "license": "GPL-2.0-or-later",
   "require": {
+    "automattic/jetpack-autoloader": "^5.0",
```

Comment:
Just chiming in here with a thought (not reason to change anything at all, just something to think about): An alternative to trying to only load one version of the dependency would be to prefix the bundled dependencies.

**Benefit:** More robust, as it prevents conflicts due to minor differences between different versions if one plugin used an API slightly differently.
**Drawback:** Code will be "unnecessarily" (at least _almost_ unnecessarily) loaded multiple times, and the initial setup is a little more tricky.

It's certainly a tradeoff. But I'd be inclined to eventually document both as reasonable alternative paths for pulling in and loading those dependencies.